src/utils/redux/slices/multisigsSlice.ts

```diff
@@ -62,6 +62,19 @@ export const multisigsSlice = createSlice({
         account.label = newName;
       }
     },
+    removeMultisigs: (state, { payload }: { payload: RawPkh[] }) => {
+      const operationIdsToRemove = state.items
+        .filter(multisig => payload.includes(multisig.address.pkh))
+        .map(multisig => multisig.pendingOperationsBigmapId);
+
+      state.items = state.items.filter(multisig => !payload.includes(multisig.address.pkh));
```

Comment:
there's no need in it because they will be cleaned up after the next data refetch which happens automatically once you remove any implicit account